test5.cpp

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+
+
+int
+main()
+{
```

Comment:
{ should almost always be at the end of the previous line  [whitespace/braces] [4]